utils/misc.go

```diff
@@ -86,3 +87,12 @@ func CleanString(s string) string {
 
 	return cleaned
 }
+
+// EncodeBase64 encodes the list of strings with Linux base64
+func EncodeBase64(strList []string) string {
+	var strBuffer bytes.Buffer
+	for _, l := range strList {
+		strBuffer.WriteString(l)
+	}
+	return base64.StdEncoding.EncodeToString([]byte(strBuffer.String()))
+}
```

Comment:
Do we need this? Why not just:

`base64.StdEncoding.EncodeToString([]byte(strings.Join(strList, ""))`